src/lib/database/goals.ts

```diff
@@ -0,0 +1,66 @@
+import sql from "$lib/database/postgres.server";
+import type { Goal } from "$lib/database/Goal";
+
+export async function getGoals(): Promise<Goal[]> {
+  const players = await sql<Goal[]>`
+      SELECT * FROM goals
+    `;
+
+  return players;
+}
+
+export async function getGoal(sheetid: number): Promise<Goal> {
+  const players = await sql<Goal[]>`
+      SELECT * FROM goals WHERE sheetid = ${sheetid}
+    `;
+
+  return players[0];
+}
```

Comment:
To be honest, I haven't been finding the general getGoals() function necessary, and when I query the database for goals by sheetId, I want to get all of the goals for the sheet instead of just one. Here's what I would suggest:
```suggestion
export async function getGoals(sheetid: string): Promise<Goal[]> {
  const goals = await sql<Goal[]>`
      SELECT * FROM goals WHERE sheetid = ${sheetid}
    `;

  return goals;
}
```